axelor-account/src/main/java/com/axelor/apps/account/web/AccountingReportController.java

```diff
@@ -228,4 +239,20 @@ public void showMoveExported(ActionRequest request, ActionResponse response) {
 
     response.setView(actionViewBuilder.map());
   }
+
+  public void setAccountTypeForAccountReport(ActionRequest request, ActionResponse response) {
+
+    AccountingReport accountingReport = request.getContext().asType(AccountingReport.class);
+    if (accountingReport.getTypeSelect() == 1000) {
+      AccountConfig accountConfig = accountingReport.getCompany().getAccountConfig();
+      if (accountConfig != null) {
+        response.setValue("accountTypeSet", accountConfig.getAdminExpAccountTypeSet());
+        response.setReadonly("accountTypeSet", true);
```

Comment:
This readonly will not work when user refresh, instead use 'readonlyIf' condition on report type select.